<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>省市名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>经度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>纬度</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>安徽省</string>
      </property>
      <property name="text">
       <string>117.250</string>
      </property>
      <property name="text">
       <string>31.833</string>
      </property>
      <item>
       <property name="text">
        <string>合肥市</string>
       </property>
       <property name="text">
        <string>117.250</string>
       </property>
       <property name="text">
        <string>31.833</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>安庆市</string>
       </property>
       <property name="text">
        <string>117.050</string>
       </property>
       <property name="text">
        <string>30.533</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>广东省</string>
      </property>
      <property name="text">
       <string>113.267</string>
      </property>
      <property name="text">
       <string>23.133</string>
      </property>
      <item>
       <property name="text">
        <string>广州市</string>
       </property>
       <property name="text">
        <string>113.267</string>
       </property>
       <property name="text">
        <string>23.133</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>深圳市</string>
       </property>
       <property name="text">
        <string>114.050</string>
       </property>
       <property name="text">
        <string>22.550</string>
       </property>
      </item>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>省市名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>经度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLon"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>纬度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLat"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnAddTop">
       <property name="text">
        <string>添加顶级节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddChild">
       <property name="text">
        <string>添加子节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelLeaf">
       <property name="text">
        <string>删除叶子节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelSubtree">
       <property name="text">
        <string>删除节点子树</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
